.buildkite/pipeline.yml

```diff
@@ -190,3 +190,17 @@ steps:
           privileged: true
           image: "rustvmm/dev:v5"
           always-pull: true
+
+  - label: "commit-format-x86"
+    commands:
+      - pytest rust-vmm-ci/integration_tests/test_commit_format.py
+    retry:
+      automatic: false
+    agents:
+      platform: x86_64.metal
+      os: linux
+    plugins:
+      - docker#v3.0.1:
+          privileged: true
```

Comment:
Does the container need root privileges for this test?

Reply:
Nope, it shouldn't.